Mathlib/Topology/IsLocalHomeomorph.lean

```diff
@@ -205,6 +205,17 @@ theorem openEmbedding_of_injective (hf : IsLocalHomeomorph f) (hi : f.Injective)
     OpenEmbedding f :=
   openEmbedding_of_continuous_injective_open hf.continuous hi hf.isOpenMap
 
+/-- A surjective open embedding is a homeomorphism. -/
+noncomputable def _root_.OpenEmbedding.toHomeomeomorph_of_surjective (hf : OpenEmbedding f)
```

Comment:
I think you don't need to assume that the embedding is open, surjectivity implies it.

Reply:
Indeed, good call. Replaced by just an embedding.